cmd/admin-config-set.go

```diff
@@ -81,37 +88,55 @@ func (u configSetMessage) JSON() string {
 }
 
 // checkAdminConfigSetSyntax - validate all the passed arguments
-func checkAdminConfigSetSyntax(ctx *cli.Context) {
-	if len(ctx.Args()) == 0 || len(ctx.Args()) > 2 {
+func checkAdminConfigSetSyntax(ctx *cli.Context) string {
+	if len(ctx.Args()) == 0 {
 		cli.ShowCommandHelpAndExit(ctx, "set", 1) // last argument is exit code
 	}
+	if len(ctx.Args()) == 1 {
```

Comment:
I think we can move this to mainAdminConfigSet

Reply:
OK